certified-connectors/Industrial App Store/README.MD

```diff
@@ -0,0 +1,104 @@
+# Industrial App Store Microsoft Power Automate Connector
+
+![IAS Connector](https://www.intelligentplant.com/datasheets/powerplatform/resources/AppStore-Automate-Connector-100px.png "Industrial App Store - Microsoft Power Automate (Flow) Connector")
+
+Intelligent Plant’s [Industrial App Store](https://appstore.intelligentplant.com) Microsoft Automate (Flow) Connector enables Microsoft’s powerful analytics and visualizations to be applied to real-time and historical process data. Seamlessly integrate plant and corporate data and share with any colleague on any device, enabling faster, better, real-time decision making. The Industrial App Store Connector connects to Intelligent Plant’s Industrial App Store, while all data **remains** secure and safe on-premises. The plant data may be centralized in a corporate data lake or globally dispersed across many sites and historians. Either way, Industrial App Store Power Automate Connector brings it all together and delivers to your fingertips.
```

Comment:
Could this please be changed to say "Intelligent Plant's Industrial App Store Microsoft Power Automate..."?

Reply:
That's updated now,
Thanks